assets/javascripts/discourse/components/chat-composer.js

```diff
@@ -217,25 +160,18 @@ export default Component.extend(TextareaTextManipulation, ComposerUploadUppy, {
       this,
       "_replyToMsgChanged"
     );
+    this.appEvents.off(
+      "upload-mixin:chat-composer-uploader:in-progress-uploads",
```

Comment:
Im mixed on the name of the event here, seems very tied to an implementation detail

Reply:
This is an event in UppyUploadMixin in core, it gets fired every time the `inProgressUploads` changes (so when one is added, successful, or removed). Maybe there is a clearer event name I can use, like `upload-status` or something? The alternative is having to hook into all the add/remove/success events from uppy directly, whereas for this composer we only care about the current state of inProgressUploads.